src/styles/blocks/header.scss

```diff
@@ -38,5 +36,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $animation-duration;
```

Comment:
```suggestion
    transition: color $animation-duration;
```

same here

Reply:
done.

sorry, when checked it's ability to work, forgot to delete "duration" from operator - and with specify "color" operator "animation-duration" didn't work

so I made incorrect conclusion, that in this case it will not work